### DISCUSSION TITLE
Setting up Firefox VPN on Gluetun

### DISCUSSION THREAD

**Discussion by adrawdet** [qdm12/gluetun#721] [Q&A] [answered]
Has anyone tested running Mozilla VPN on Gluetun? 

Was hoping it would be similar to Mullvad since they're renting the servers from them, but can't seem to find much info regarding Mozilla VPN.

**Comment by qdm12** [comment, marked as answer]
I have no idea. If they propose Wireguard or OpenVPN configuration files, you can try it out with this projects using the `custom` VPN provider in this repo (see the github wiki).

**Reply by adrawdet** [reply]
Doesn't look like they give you access to that information just yet 🥲. Will have to try it another time once they start to add more options/features
